dot/parachain/availability-store/availability_store.go

```diff
@@ -156,6 +188,91 @@ func (asb *availabilityStoreBatch) reset() {
 	asb.pruneByTime.Reset()
 }
 
+var (
+	TestChunk1 = ErasureChunk{
```

Comment:
These test fixtures shouldn't be here.  Move to a separate package if you want to share these fixtures between tests in different packages.

Reply:
These have been moved to test availability store package.